pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "ftp://ftp.mozilla.org/pub/thunderbird/releases/${verName}/source/thunderbird-${verName}.source.tar.bz2";
-    sha1 = "90e18f8ecccdaf1ee39493223a7e3ad8b3b7bede";
+    sha1 = "7bb0c85e889e397e53dcbcbd36957dbd7c8c10bd";
```

Comment:
Add a comment with a reference to http://archive.mozilla.org/pub/thunderbird/releases/38.1.0/SHA1SUMS
And you should prefer the sha512, now.